tests/root-crate-path/src/main.rs

```diff
@@ -1,4 +1,4 @@
-#[derive(Clone, PartialEq, ::prost::Message)]
+#[derive(Clone, PartialEq, Eq, ::prost::Message)]
```

Comment:
This should get an allow with the new version of prost right?

Reply:
This looks like my mistake. I'll fix this.

Reply:
Fixed.